projects/storefrontlib/src/shared/config/qualtrics-config.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
```

Comment:
No need for this file, please remove it.

Reply:
we need this file because it's how the config will work, if there's no object in app.module, then return {}, else they would need to add a projectId